plasmapy/utils/decorators/helpers.py

```diff
@@ -142,3 +142,51 @@ def wrapper(*args, **kwargs):
         wrapper.__signature__ = inspect.signature(f)
 
     return wrapper
+
+
+def add_lite(flite, attrs=None, scope=None):
```

Comment:
Maybe "register_lite", come to think of it? I think "register" has a pretty common meaning for decorators.

Reply:
When I think "register" I'm think it's registering the functionality with some kind of table/database.  How about `bind_lite`?  That's closer to what it is actually doing.

Reply:
That sounds great!